scalikejdbc-core/src/main/scala/scalikejdbc/TypeConverter.scala

```diff
@@ -0,0 +1,159 @@
+package scalikejdbc
+
+import java.io.InputStream
+import java.sql.{ PreparedStatement, ResultSet }
+import scalikejdbc.UnixTimeInMillisConverterImplicits._
+
+trait TypeConverter[A] extends TypeBinder[A] with ParameterBinderFactory[A] { self =>
```

Comment:
The handy way is very nice. But I don't prefer the name `Converter`. I may miss the origin of the name but anyway I'm not willing to accept this name. 

The reason is that I think we should have at least `Bind` in the name because this module is at once a `TypeBinder` and a `ParameterBinderFactory`.

How about `Binders`?